.github/CHANGELOG.md

```diff
@@ -483,8 +659,9 @@
 
 This release contains contributions from (in alphabetical order):
 
-Thomas Bromley, Olivia Di Matteo, Theodor Isacsson, Josh Izaac, Christina Lee, Alejandro Montanez,
-Steven Oud, Chase Roberts, Maria Schuld, Antal Száva, David Wierichs, Jiahao Yao, Sankalp Sanand.
+Juan Miguel Arrazola, Thomas Bromley, Olivia Di Matteo, Theodor Isacsson, Josh Izaac, Christina Lee,
+Alejandro Montanez, Nathan Killoran, Steven Oud, Chase Roberts, Sankalp Sanand, Maria Schuld, Antal
+Száva, David Wierichs, Jiahao Yao.
```

Comment:
Went through all PRs since last release, but might have missed some contributors that code reviewed but didn't make a PR! Let me know if anyone is missing.